Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -329,7 +330,7 @@ def set_preprocessor(self, preproc):
     def handleNewSignals(self):
         """Reimplemented from OWWidget.handleNewSignals."""
         self._update_class_selection()
-        self.apply()
+        self.commit()
```

Comment:
It's sometimes _very_ useful to be able to disable flow propagation in slow Test & Score. Imagine I want to connect multiple stuff and only run the simulation at the end, not wait 20 stupendous seconds between my every action.

Whether the widgets always auto apply when the input signal has new data is something that should be uniformly determined and in the guidelines?